### ISSUE TITLE
Not sure if this is the correct place

### ISSUE THREAD

**Issue by wolftone1916** [kimai/kimai#204]
So I am not new to web design but and new to web apps and the newer back end operations. I am trying to get v2 installed and my site is hosted on inmotion hosting. Is this something they have to do for me to add it to my site much like v1 or am I just totally lost about how to get it installed with Git and Composer?

It was listed in their apps  to add and I would just need to click install. I probably sound dumb with this but I would REALLY love to use this as the company I am with would love to use it and continue to use it as a way to record reps out in the field and their times of work. I appreciate any feedback on this and if I need to get in contact with inmotion to add V2 I will...

**Comment by kevinpapst** [comment]
Hi!
If you want to have a one-click install, then inmotion has to do it on their side.
Otherwise you can install Kimai v2 if you have SSH access to your server. If you only have FTP, then probably this might help: https://github.com/kevinpapst/kimai2/issues/199#issuecomment-404102205

**Comment by wolftone1916** [comment]
I have access to the cpanel of my site being that I'm the owner. However, I looked at connecting via SSH and it looks like there MIGHT be a way using a console program. I might end up contacting them to update to V2 though so it's a one click install then. Thank you for the prompt response. This add-on is absolutely class.

**Comment by kevinpapst** [comment]
Thanks for your feedback, very much appreciated!

I will close this ticket, just open a new one for further question.

**Comment by wolftone1916** [comment]
Not sure if this reopens it but I am curious. I am logged in via ssh right now and see:

[myusername@ecbiz224 ~}$ 

I'm honestly at a loss as what to do next. I tried the first part of the development install and then the production but neither were able to do it. I guess I'm not sure once I'm logged in via SSH what the next step is...

**Comment by kevinpapst** [comment]
You have to go to your webserver root (probably at `/var/ww/`) and execute the installation steps there. Then you need to configure your web server to use the directory `/var/ww/kimai2/public` as document root.
But that is highly system specific. If you have no knowledge about the Linux bash, it might be easier to execute everything locally and then upload the files with FTP.

**Comment by wolftone1916** [comment]
Thank you, I meant to respond sooner but I didn't get the notification about this. I was finally able to get the install to start but I found my php version was not 7.1 so it wasn't successful. I'm looking to upgrade to that to do this, I just need to make sure the other apps I'm using are compatible with 7.1. As well I contacted softapulous which is who is holding your app for us to use about getting the v2 included on the this. The only thing that they said why not right now is that it's not a stable/full release. Once it is, they said they would be more than happy to put it in the list for a 1 click install.
